packages/react-server-module-tagger/index.js

```diff
@@ -0,0 +1,129 @@
+var isWindows = ('win32' === process.platform);
+
+/**
+ * A util function for tagging modules.  Expects to find data on its prototype
```

Comment:
It expects to find data in its context, right?  Doesn't need to be on the prototype.